### DISCUSSION TITLE
47 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by aryll** [AzureCommunityPL/group#273] [Wydarzenia]
Już 12 maja będziemy mieli okazję poznać trzy fascynujące sesje od trzech interesujących prelegentów z trzech różnych perspektyw! Dołącz do spotkania po potrójną dawkę wiedzy!

Agenda:
18:00-18:05 Powitanie - Paweł Haracz
18:05-18:35 Który magazyn danych Azure Search lubi najbardziej? - Maciej Gos
18:35-18:50 15 min z Azure Governance - krótka rozprawka o standardzie nazewniczym - Kamil Więcek
18:50-18:35 Azure Functions - fight for minimal downtime - Krzysztof Madej

Link do stream: https://www.youtube.com/watch?v=0KL8qgj3Wag
Wydarzenie na meetup: https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/277587620/

![Three-Frame Tea Set Photo Facebook Cover (4)](https://user-images.githubusercontent.com/5950108/116380552-5becf400-a814-11eb-89b8-bfd592700189.png)

----------------------------------------------------------------
🧍 Maciej Gos

Architekt/Team Leader oraz pasjonat chmury (nie tylko od Microsoft) od lat programista .NET i nie tylko
https://www.linkedin.com/in/maciejgos/
https://blog.maciejgos.com

🎙️ Sesja: Który magazyn danych Azure Search lubi najbardziej?

Z jakimi magazynami danych Azure Search potrafi pracować oraz na co należy zwracać uwagę przy ich wykorzystaniu. Co może nas zaskoczyć?

----------------------------------------------------------------
🧍 Kamil Więcek

Orędownik orchestracji. Obecnie konsultant Azure pomagający klientom w uniknięciu błędów podczas adaptacji chmury.
https://www.linkedin.com/in/kamil-wiecek
https://www.kamilwiecek.com/

🎙️ Sesja: 15 min z Azure Governance - krótka rozprawka o standardzie nazewniczym

Nazywanie zasobów towrzyszy nam od zawsze. Standard nazewniczy pomagał nam w utrzymaniu porządku w środowisku. Dlaczego w Azure jest łatwiej? Co zrobić, jeżeli nie ma go do tej pory? Kiedy jest niezbędny, a kiedy można żyć bez niego? Jak go wdrożyć i wymusić w podejśćiu 'as a code'? Na tych kilka pytań chętnie odpowiem podczas tej krótkiej sesji.

----------------------------------------------------------------
🧍 Krzysztof Madej

Mąż, tata oraz dev udający opsa :) Z Azurem zaznajomiony od półtora roku, a wcześniej to .NET i trochę frontu. Na codzień pracuję jako DevOps Lead, a po godzinach wrzucam coś na StackOverflow i na swojego bloga (to rzadziej - zawsze jednak brakuje czasu, czyż nie?). To właśnie tak to przebiegła moja przemiana z czystego .Net na Azure DevOps'a.
https://www.linkedin.com/in/krzysztof-madej/
http://thecodemanual.pl/

🎙️ Sesja: Azure Functions - fight for minimal downtime

Azure Functions podobnie jak Azure App Services dają możliwość skorzystania ze deployment slots, aby klient nie odczuł samego wdrożenia. Tyle czy same sloty to wszystko? W sesji przedstawię co należy zrobić aby zminimalizować downtime i tym samym zapewnić maksymalne SLA dla naszych funkcji

**Comment by mifurm** [comment]
dobre spotkanie się szykuje, dużo fajnych, nowych twarzy.
